config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  root "works#homepage", as: "home"
+
+  post "works/:id/vote", to: "works#vote", as: "vote"
+  resources :works #, except: [:destroy]
+
+  get "/login", to: "users#login_form", as: "login"
+  post "/login", to: "users#login"
+  post "/logout", to: "users#logout", as: "logout"
```

Comment:
You have a route and controller action set up to log out the user, but your site doesn't have a link or button pointing at that route so there is no way for the user to do it.